src/utils/filters/qs.ts

```diff
@@ -0,0 +1,11 @@
+const paramsToRemove = ["activeTab", "action", "sort", "asc"];
+
+export const prepareQs = (searchQueary: string): URLSearchParams => {
+  const qs = new URLSearchParams(searchQueary);
+
+  paramsToRemove.forEach(param => {
+    qs.delete(param);
+  });
+
+  return qs;
+};
```

Comment:
**issue**: I remember there is already function that does exactly this

Reply:
yeah we have parse from "qs" library and I remember it's a bit different than URLSearchParams, please double check. Ideally we should use URLSearchParams everywhere, but it might not be that easy